app/models/course.rb

```diff
@@ -290,6 +293,10 @@ def initially_refreshed?
     refreshed_at.nil?
   end
 
+  def assistant?(user)
+    assistants.include? user
```

Comment:
I'm wondering whether this is basically `(select * from assistants where course_id=...).to_a.include?` or if this is performed with actual sql querty. with bigger courses this might become really ineficient